src/services/image/resize.rs

```diff
@@ -0,0 +1,45 @@
+use crate::utils::Result;
+use image::FilterType;
+use std::fs::File;
+use std::io::{BufRead, BufReader};
+use std::path::PathBuf;
+
+struct Format {
+    name: &'static str,
+    x_height: u32,
+    y_height: u32,
+}
+
+static FORMATS: &'static [Format] = &[
+    Format {
+        name: "thumbnail",
+        x_height: 120,
+        y_height: 120,
+    },
+    Format {
+        name: "medium",
+        x_height: 480,
+        y_height: 480,
+    },
+    Format {
+        name: "large",
+        x_height: 1080,
+        y_height: 1080,
+    },
+];
+
+pub fn call(original_file: &PathBuf, file_ext: &str) -> Result<Vec<PathBuf>> {
```

Comment:
Extension of comment from `src/resolvers/medium.rs:19`: maybe we should keep the internal naming structure of the services module, because it kinda makes sense when viewing the service source code; but instead of making `call` public, maybe we should do something like `pub use resize::call as resize` in `services/image`, so that the external API to the services don't get caught up in the internal formatting itself.